shared/desktop/yarn-helper.js

```diff
@@ -177,64 +177,29 @@ const commands = {
 }
 
 function postInstall () {
-  if (process.platform === 'win32') {
```

Comment:
don't have to deal with the shared code being a symlink. renderer/fonts isn't being fixed up but i think that'd only affect devving on windows. we can revisit this